src/aiida/cmdline/commands/cmd_init.py

```diff
@@ -0,0 +1,104 @@
+)
+def verdi_init(directory, from_archive):
+    """Initialize a new AiiDA instance.
+
+    The instance is initialized in DIRECTORY, which defaults to the currrent working directory. A profile is
+    automatically created using the `core.sqlite_dos` storage backend and the localhost is configured as a computer.
+    """
+    import tempfile
+    import warnings
+
+    from aiida.common import exceptions
+    from aiida.manage import get_manager
+    from aiida.manage.configuration import create_profile, get_config, load_profile, reset_config, settings
+    from aiida.orm import Computer
+    from aiida.storage.sqlite_dos import SqliteDosStorage
+    from aiida.storage.sqlite_zip import SqliteZipBackend
+
+    # Unload the profile that was loaded by default and reset the configuration.
+    get_manager().unload_profile()
+    reset_config()
+
+    dirpath_config = directory / settings.DEFAULT_CONFIG_DIR_NAME
+
+    if dirpath_config.is_dir():
+        echo.echo_critical(f'A configuration directory already exists in `{directory}`.')
+
+    with warnings.catch_warnings():
+        warnings.simplefilter('ignore')
+        settings.set_configuration_directory(dirpath_config)
+
+    echo.echo_success(f'Initialized new AiiDA instance in `{dirpath_config}`.')
+
+    # Now load the config causing the configuration file to be created.
+    config = get_config(create=True)
+    dirpath_loaded = pathlib.Path(config.filepath).parent
+    assert dirpath_loaded == dirpath_config, f'Filepath of loaded config `{dirpath_loaded}` != `{dirpath_config}`.'
+
+    if from_archive:
+        storage_cls = SqliteZipBackend
+        storage_config = {
+            'filepath': str(from_archive),
+        }
+    else:
+        storage_cls = SqliteDosStorage  # type: ignore[assignment]
+
+    try:
+        profile = create_profile(
+            config,
+            name='init',
+            email='aiida@localhost',
+            storage_cls=storage_cls,
+            **storage_config,
+        )
+    except (ValueError, TypeError, exceptions.EntryPointError, exceptions.StorageMigrationError) as exception:
+        echo.echo_critical(str(exception))
+
+    echo.echo_success(f'Created new profile `{profile.name}`.')
+
+    config.set_option('runner.poll.interval', 1, scope=profile.name)
+    config.set_default_profile(profile.name, overwrite=True)
+    config.store()
+
+    if not from_archive:
+        echo.echo_info(f'Loaded newly created profile `{profile.name}`.')
+        load_profile(profile.name, allow_switch=True)
+
+        computer = Computer(
+            label='localhost',
+            hostname='localhost',
+            description='Localhost automatically created by `verdi blitz`',
+            transport_type='core.local',
+            scheduler_type='core.direct',
+            workdir=str(pathlib.Path(tempfile.gettempdir()) / 'aiida_scratch'),
+        ).store()
+        computer.configure(safe_interval=0.0)
+        computer.set_minimum_job_poll_interval(0.0)
```

Comment:
We've just been discussing with @giovannipizzi, @unkcpz and others about a pathological performance of AiiDAlab demo server, and I suggested to look at various poll intervals. I haven't dug too deep or done extensive testing, but I did increase the `job_poll_interval` in my deployment. So I wonder if a safer default is warranted here.

```suggestion
        computer.set_minimum_job_poll_interval(0.1)
```

Reply:
I also vote to use a large interval. I indeed found that the daemon used a lot CPU in the AiidAlab demo server.

Reply:
Can only be set to integers, so set it to 1